src/test/java/de/tum/in/www1/artemis/iris/IrisHestiaIntegrationTest.java

```diff
@@ -47,11 +48,13 @@ void initTestCase() {
     void updateSolutionEntriesOnSaving() throws Exception {
         addCodeHints();
 
-        irisRequestMockProvider.mockRunResponse(dto -> {
-            assertThat(dto.settings().authenticationToken()).isNotNull();
-
-            pipelineDone.set(true);
-        });
+        /*
+         * irisRequestMockProvider.mockRunResponse(dto -> {
+         * assertThat(dto.settings().authenticationToken()).isNotNull();
+         * pipelineDone.set(true);
+         * });
+         */
+        fail("This test is not yet implemented. Implement it and remove the fail call.");
```

Comment:
This seems to be in a work in progress state 😅

Reply:
Yes. This entire thing here is completely WIP.